rosidl_generator_cpp/msg/PrimitivesSomeDefault.msg

```diff
@@ -0,0 +1,17 @@
+float32 float32_value 1.125
+float64 float64_value 2.4
+uint64 uint64_value
+string string_value "bar"
+float32[10] float32_arr
+float64[3] float64_arr [8.5, 1.2, 3.4]
+float64[2] float64_arr2 [99.0, 2.3]
+string[3] string_arr
+# TODO(clalancette): default values for strings don't seem to work; why?
```

Comment:
What about this todo? Should this be resolved before merging this (at least figuring out the "why")?

Reply:
Maybe update the TODO text then.

Reply:
Done in 114078e